src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -56,19 +66,51 @@ class Lambda {
     dotcomSurrogateKey
   }
 
-  private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
-    if (sendFastlyPurgeRequest(contentId, purgeType, serviceId, surrogateKey, fastlyApiKey))
-      sendAmpPingRequest(contentId)
-    else
-      false
+  // Left[Id] for Delete, Right[Id] for Update
+  def decidePurgeType: PartialFunction[Event, Either[Id, Id]] = {
```

Comment:
It looks like you're using a different function syntax for the new functions in this file - maybe best to keep it the same as all the other functions and if you want to change it to refactor them together? Just so there aren't two code styles going on in the same project

Reply:
Good shout, thanks Phil